src/components/multi-action-button/multi-action-button.component.tsx

```diff
@@ -198,29 +217,29 @@ export const MultiActionButton = ({
       setTimeout(() => {
         additionalButtons.current[0]?.current?.focus();
       }, 0);
+    } else if (Events.isEscKey(ev)) {
+      ev.preventDefault();
+      setShowAdditionalButtons(false);
     }
   };
 
-  const focusMainButton = () => {
+  const handleFocusMainButton = () => {
     isMainButtonFocused.current = true;
     if (isFocusedAfterClosing.current) {
       isFocusedAfterClosing.current = false;
```

Comment:
as far as I can tell, the isFocusedAfterClosing ref is now no longer needed at all - its only purpose seemed to be here, to avoid showing the buttons if it was true. And we're now not showing the buttons in any case. I think this code can be cleaned up by removing the ref entirely.